### PR TITLE
Added CONTRIBUTING.md with the changes requested in PR #33

### PR DESCRIPTION
**Pull request by SKalt**
I've included the work done by @avirlrma in #33 with the changes requested by @pombredanne.  I'll be around most nights of this week for any edits as needed.

<details open><summary>Sign-off</summary>
I sign off on the DCO.
</details>

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+Thank you for your interest in `tools-python`. The project is open-source software, and bug reports, suggestions, and most especially patches are welcome.
+
+## Issues
```
Comment (omi10859) [inline CONTRIBUTING.MD]: we can also include issue template if needed or available

Comment (SKalt) [reply CONTRIBUTING.MD]: I agree. Want to add in a separate issue/PR pair?

**Comment by SKalt** [comment]
I'm going to fix the  `-s` issue tonight. I may also document it in CONTRIBUTING.md.

File: CONTRIBUTING.MD [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Patches
+
+The source code for `tools-python` is hosted on [github.com/spdx/tools-python](https://github.com/spdx/tools-python). Please review [open pull requests](https://github.com/spdx/tools-python/pulls) before committing time to a substantial revision. Work along similar lines may already be in progress.
+
+To submit a patch via GitHub, fork the repository, create a topic branch from `master` for your work, and send a pull request when your local tests pass (`./setup.py test`).
+
+## Licensing
+
+However you choose to contribute, please confirm in a comment to your pull request that you license your contributions under the terms of [the Developer Certificate of Origin](https://developercertificate.org/).
```
Comment (pombredanne) [inline CONTRIBUTING.MD]: The way this should be done is with a `Signed-off-by: name <foo@email.com>`  commit message instead... and this is what is enforced by the bot too, btw ;)

Comment (SKalt) [reply CONTRIBUTING.MD]: Noticed.  Is there a method you'd recommend to locally verify your branch is properly signed-off?  It looks like `rebase --signoff` might be the right way to fix missing signoffs; got an opinion?

File: CONTRIBUTING.MD (hunk 1 above)
Comment (omi10859) [reply CONTRIBUTING.MD]: @pombredanne do we have any issue template or do we even need one?

Comment (pombredanne) [reply CONTRIBUTING.MD]: we can have one. Simpler is better though, and this has never been an issue for now. So this can wait

**Pull request merged**